spec/models/transaction_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Transaction, type: :model do
+  describe "#foobar" do
+    it "should ..." do
+      # setup
+      # exercise
+      # verify
+    end
```

Comment:
there should not be ends in each "it" I think. I received an unexpected "end" error when I tried this specific format

Reply:
joke tama naman pala. i just forgot to put a "do"